cli/src/main/java/com/devonfw/tools/ide/environment/AbstractEnvironmentVariables.java

```diff
@@ -161,16 +179,31 @@ public EnvironmentVariables resolved() {
   @Override
   public String resolve(String string, Object src) {
 
-    return resolve(string, src, 0, src, string);
+    return resolve(string, src, 0, src, string, EnvironmentVariablesType.CONF);
   }
 
-  private String resolve(String value, Object src, int recursion, Object rootSrc, String rootValue) {
+  public String resolve(String value, Object src, int recursion, Object rootSrc, String rootValue) {
```

Comment:
```suggestion
  private String resolve(String value, Object src, int recursion, Object rootSrc, String rootValue) {
```

Why should we make this internal method public? And if you do, you should add JavaDocs.

Reply:
Thanks, I changed it.